src/client_comms.js

```diff
@@ -39,8 +39,12 @@ class ClientComms {
     this.io = new Server(
       expressServer,
       {
-        pingInterval: PING_INTERVAL_MS,
-        pingTimeout: PING_TIMEOUT_MS
+        cors: {
+          origin: 'http://127.0.0.1:8080', // Replace with your actual client's origin
```

Comment:
This seems really fragile if it must be configured at compile time for each specific environment.
Also, why is the port number not RQ_PARAMS.SERVER_PORT_NUMBER or a new constant in params.js?

Reply:
a new constant would be fine.
a * would open it for anywhere.